sdk/communication/Azure.Communication.CallingServer/src/Models/DtmfTone.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.CallingServer
+{
+    /// <summary>
+    /// The possible Dtmf Tones.
```

Comment:
"Tones"? Is this a collection or is it a single "Tone"?

Reply:
It is a single tone.